### PR TITLE
OCPQE-22055 bump ec to rc for 4.16 upgrade jobs

### PR DESCRIPTION
**Pull request by dis016**
[OCPQE-22055](https://issues.redhat.com//browse/OCPQE-22055) Bump images from ec to rc for 4.16 jobs

### CODE REVIEW

**Comment by dis016** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.16-azure-mag-ipi-fullyprivate-f14

**Comment by dis016** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.16-azure-mag-ipi-fullyprivate-f14

**Comment by dis016** [comment]
`INFO[2024-05-16T06:40:23Z] Importing release 4.16.0-rc.1 created at 2024-05-10 16:23:41 +0000 UTC with 186 images to tag release:target ... 
INFO[2024-05-16T06:40:37Z] Importing release 4.16.0-rc.0 created at 2024-05-06 20:34:08 +0000 UTC with 187 images to tag release:latest ... `

**Comment by dis016** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.16-azure-mag-ipi-fullyprivate-f14

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml [hunk 1]
```diff
@@ -48,16 +48,12 @@ releases:
       relative: 1
       version_bounds:
         lower: 4.16.0-0
-        stream: 4-dev-preview
         upper: 4.17.0-0
   target:
-    prerelease:
+    release:
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml:53]: From openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.15.yaml, sounds like it is supposed to use `version_bounds` here too.

**Comment by jianlinliu** [comment]
Shall we also update `openshift-openshift-tests-private-release-4.16__amd64-rollback-ec.yaml` to use stable build here?

**Comment by dis016** [comment]
> Shall we also update `openshift-openshift-tests-private-release-4.16__amd64-rollback-ec.yaml` to use stable build here?

yes we can

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml (hunk 1 above)
Comment (dis016) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml:53]: @jianlinliu ok, i will update accordingly.

Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml:53]: Just want to know what's the difference between `version_bounds` and `channel`. Maybe a release goes to release controller a bit earlier than cincinnati channel, but not too much time difference.

Comment (dis016) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml:53]: @shellyyang1989 yes, when i tested there was no difference in the output. we can fetch n-1 version using prerelease and relative, whereas in release relative option is not available.

**Comment by dis016** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.16-azure-mag-ipi-fullyprivate-f14

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml (hunk 1 above)
Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml]: Got it. Thank you!

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-ec.yaml [hunk 2]
```diff
@@ -68,15 +68,13 @@ releases:
       relative: 1
       version_bounds:
         lower: 4.16.0-0
-        stream: 4-dev-preview
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-ec.yaml:71]: I think we also need to rename the yaml file to replace `ec` with `stable`.

**Comment by jianlinliu** [comment]
@dis016 the PR is ready to move forwards? If yes, pls add "pr-rehearse ack" label, and ask related OWNER to approve it for the merge.

**Comment by jianlinliu** [comment]
cc @liangxia to review

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.16-azure-ipi-proxy-tp-f28

**Comment by jianlinliu** [comment]
@dis016 from https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/52084/pull-ci-openshift-release-master-core-valid/1793546855150260224, sounds like a conflicts, need a rebase.

**Comment by dis016** [comment]
reopening in https://github.com/openshift/release/pull/52418

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml [hunk 3]
```diff
@@ -10312,17 +10312,6 @@ periodics:
     job-release: "4.16"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-f7-netobserv
-  reporter_config:
-    slack:
-      channel: '#team-qe-netobserv'
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml:10317]: @dis016 Those removal are unexpected

**Pull request closed without merging**